src/test/java/org/fundacionjala/api/stepdefs/RequestSteps.java

```diff
@@ -1,5 +1,6 @@
 package org.fundacionjala.api.stepdefs;
 
+/*import io.cucumber.java.en.And;*/
```

Comment:
Please consider deleting this commented line.

Reply:
done